portfolio/src/main/webapp/nav.html

```diff
@@ -0,0 +1,17 @@
+<div>
+    <h2 class="navText">Roddy's Portfolio</h2>
+</div>
+<div id="navLinks">
+    <a href="index.html" class="navLink">
+        <h4 class="navText">Home</h3>
+    </a>
+    <a href="about.html" class="navLink">
+        <h4 class="navText">About</h3>
+    </a>
+    <a href="" class="navLink">
+        <h4 class="navText">Projects</h3>
+    </a>
+    <a href="" class="navLink">
+        <h4 class="navText">Blog</h3>
+    </a>
+</div>
```

Comment:
please add a new line at the end of the file

Reply:
Will do, but is it because of style or is it more semantic?

Reply:
It is just style.